modules/channel-web/src/backend/api.ts

```diff
@@ -630,19 +627,18 @@ export default async (bp: typeof sdk, db: Database) => {
   privateRouter.post(
     '/conversations/:id/messages/delete',
     asyncMiddleware(async (req: ChatRequest, res: Response) => {
-      const botId = req.params.botId
-      let conversationId = req.params.id
+      const conversationId = req.params.id
       const { userId } = req.body
 
-      conversationId = parseInt(conversationId)
-
-      if (!(await db.isValidConversationOwner(userId, conversationId, botId))) {
+      const conversation = await req.messaging.conversations.get(conversationId)
```

Comment:
This doesn't work... The messaging client is added in `assertUserInfo` which is only added in methods in the unsecured router